analysis/src/Event.cpp

```diff
@@ -186,6 +193,68 @@ PhysObjects Event::GetGoodElectrons(TH1D *cutFlowHist)
   return physObjects.at(kGoodElectron);
 }
 
+PhysObjects Event::GetGoodMuons(TH1D *cutFlowHist)
+{
+  if(physObjectsReady.at(kGoodMuon)) return physObjects.at(kGoodMuon);
+  
+  physObjects.at(kGoodElectron).clear();
```

Comment:
Uu, please don't clear my good electrons when selecting good muons! ;) Also below, please change to kMuon/kGoodMuon